doc/README.md

```diff
@@ -75,6 +75,7 @@ gridstack.js API
   * a string (ex: '100px', '10em', '10rem', '10%')
   * 0 or null, in which case the library will not generate styles for rows. Everything must be defined in CSS files.
   * `'auto'` - height will be calculated to match cell width (initial square grid).
+- `row` - number of rows. This is a shortcut of writting `minRow: sameValue, maxRow: sameValue`.
```

Comment:
great, but all those need to be alphabetized please. also `row` should say something like:

`row` - fixed number of rows which prevents the grid from changing as items are added/removed/moved, Shortcut for setting `minRow: N, maxRow: N`. default is `0` no constrain.